hack/setup-bob-4.4.sh

```diff
@@ -0,0 +1,55 @@
+if [[ -z "$KUBECONFIG" ]]; then
+  echo "please export KUBECONFIG env var"
+  exit 1
+fi
+
+if [[ -z "$KUBEADMIN_PASSWORD" ]]; then
+  echo "please export KUBEADMIN_PASSWORD env var"
+  exit 1
+fi
+
+## check if it is a 4.x cluster
+if oc get oauth cluster > /dev/null;  then
+  echo "check 4.x cluster verified"
+else
+  echo "not a 4.x cluster, please export correct kubeconfig";
+  exit 1
+fi
+
+htpasswd -c -B -b users.htpasswd bob bob
+htpasswd -B -b users.htpasswd alice alice
+oc create secret generic htpass-secret --from-file=htpasswd=./users.htpasswd -n openshift-config
+oc patch oauth cluster --type='json' -p '[ { "op": "replace", "path": "/spec", "value": {"identityProviders": [{"htpasswd": {"fileData": {"name": "htpass-secret"}}, "mappingMethod": "claim", "name": "bob_httpassd_provider", "type": "HTPasswd"}]}}]'
+sleep 2
+oc login -u bob -p bob
```

Comment:
We could avoid mutating the existing KUBECONFIG and requesting the KUBEADMIN password for re-login if we make a temporary copy of the active KUBECONFIG. 

https://github.com/konveyor/mig-controller/blob/master/hack/controller-sa-login.sh#L10-L12

Reply:
+1, this looks exactly like what I needed to get rid of KUBEADMIN_PADDWORD, will modify